tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -289,47 +290,96 @@ def pyramid_request(self, pyramid_request):
+            "TEST_RESOURCE_LINK_ID",
+            "TEST_DOCUMENT_URL",
+        )
+
+    def test_it_configures_via_url(self, context, pyramid_request, via_url):
+        self.make_request(context, pyramid_request)
+
+        via_url.assert_called_once_with(pyramid_request, "TEST_DOCUMENT_URL")
+        assert context.js_config["urls"]["via_url"] == via_url.return_value
+
+    def test_it_configures_frontend_grading(
+        self, context, pyramid_request, frontend_app, via_url, ModuleItemConfiguration,
+    ):
+        self.make_request(context, pyramid_request)
+
+        frontend_app.configure_grading.assert_called_once_with(
+            pyramid_request, context.js_config
+        )
+
+    @pytest.fixture
+    def pyramid_request(self, pyramid_request):
+        pyramid_request.parsed_params = {
+            "document_url": "TEST_DOCUMENT_URL",
+            "resource_link_id": "TEST_RESOURCE_LINK_ID",
+            "tool_consumer_instance_guid": "TEST_TOOL_CONSUMER_INSTANCE_GUID",
+        }
+
+        return pyramid_request
+
+
 class TestUnconfiguredBasicLTILaunch:
     def test_it_sets_the_right_javascript_config_settings(
-        self, BearerTokenSchema, bearer_token_schema, context, pyramid_request
+        self, bearer_token_schema, context, pyramid_request
     ):
-        pyramid_request.params[
-            "custom_canvas_api_domain"
-        ] = "TEST_CUSTOM_CANVAS_API_DOMAIN"
+        pyramid_request.params.update({"some_random_rubbish": "SOME_RANDOM_RUBBISH"})
+
         pyramid_request.registry.settings["google_client_id"] = "TEST_GOOGLE_CLIENT_ID"
         pyramid_request.registry.settings[
             "google_developer_key"
         ] = "TEST_GOOGLE_DEVELOPER_KEY"
 
         BasicLTILaunchViews(context, pyramid_request).unconfigured_basic_lti_launch()
 
-        BearerTokenSchema.assert_called_once_with(pyramid_request)
-        bearer_token_schema.authorization_param.assert_called_once_with(
-            pyramid_request.lti_user
-        )
         assert context.js_config == {
             "mode": "content-item-selection",
             "enableLmsFilePicker": False,
             "formAction": "http://example.com/module_item_configurations",
-            "formFields": {
-                "authorization": bearer_token_schema.authorization_param.return_value,
-                "resource_link_id": "TEST_RESOURCE_LINK_ID",
-                "tool_consumer_instance_guid": "TEST_TOOL_CONSUMER_INSTANCE_GUID",
-                "oauth_consumer_key": "TEST_OAUTH_CONSUMER_KEY",
-                "user_id": "TEST_USER_ID",
-                "context_id": "TEST_CONTEXT_ID",
-            },
+            "formFields": Any.dict(),
             "googleClientId": "TEST_GOOGLE_CLIENT_ID",
             "googleDeveloperKey": "TEST_GOOGLE_DEVELOPER_KEY",
             "customCanvasApiDomain": context.custom_canvas_api_domain,
             "lmsUrl": context.lms_url,
             "urls": {},
         }
 
+        # Test param pass through
```

Comment:
I've pulled off the params to make it easier to see what's failing. This way we get separate assertion failures for the root elements and the params.